src/main/java/org/testobject/appium/junit/TestObjectAppiumSuite.java

```diff
@@ -159,10 +151,18 @@ public TestObjectAppiumSuite(Class<?> clazz) throws InitializationError {
 			Set<DataCenterSuite> dataCenterSuites = getDataCenterSuites();
 
 			this.perDeviceRunners = toRunners(clazz, dataCenterSuites);
+		}
+		runAllDevicesAtOnce(config);
+	}
 
-			this.setScheduler(new ThreadPoolScheduler(perDeviceRunners.size(), testObjectTimeout, config.timeoutUnit()));
+	private void runAllDevicesAtOnce(TestObject config) {
+		int numRunners = perDeviceRunners.size();
+		int timeout = Env.getTimeout()
+			.map(Integer::parseInt)
+			.orElseGet(config::timeout);
```

Comment:
This doesn't look too long to put it in 3 lines,
but feel free to keep it if you think it is more readable

Reply:
Having each operation on a different line improves readability. The reader needs to split the statement into the single operations. For different lines this is pretty easy but for a single line the reader has to first look for the dots and then separate the statement by its own.